src/sinks/util/retries.rs

```diff
@@ -133,7 +133,7 @@ where
                         None
                     }
                 } else if error.downcast_ref::<Elapsed>().is_some() {
-                    warn!("Request timed out.");
+                    warn!("Request timed out. If this happens often while the events are actually reaching their destination, try decreasing `batch.max_bytes` and/or using `compression` if applicable. Alternatively `request.timeout_secs` can be increased.");
```

Comment:
The length of this makes me wonder if we shouldn't instead include a link to a stable documentation page or something.

Reply:
Good idea. I see the link being included as a label to warnings and errors that lead to a reference style page with entry for each of them and possible causes and fixes for them. I don't expect too many of them, and they can be added ad hoc while being really useful and user friendly. cc @jszwedko 

But that can be added later, so I'll be merging this as is.